src/components/Button.js

```diff
@@ -1,17 +1,28 @@
 import React from 'react';
 import PropTypes from 'prop-types';
 
-const Button = ({ name }) => (
-  <button type="button" id={name} className="btn btn-info">
+import styles from './styles.module.css';
+
+const Button = ({ name, wide, color }) => (
+  <button
+    type="button"
+    className={`${styles.button}
+                  ${color ? null : styles.button_gray}
+                  ${wide ? styles.button_double : null}`}
+  >
     {name}
   </button>
 );
 
 Button.defaultProps = {
   name: null,
+  color: false,
+  wide: false,
 };
 Button.propTypes = {
   name: PropTypes.string,
```

Comment:
- We can make the name props required, this makes more sense based on the context of the Calculator application since we do not want buttons without a name. This way we do not need to declare a default value for it

```JS
Button.propTypes = {
  name: PropTypes.string.isRequired,
```

Reply:
Thank you for that Torres, it makes much sense. Have a great day!